style.css

```diff
@@ -416,6 +451,26 @@ textarea:hover {
   z-index: 500;
 }
 
+.menuitem.active {
+  color: #fff;
+  margin: 5vh;
+  padding-bottom: 3vh;
+  border-bottom: solid 2px;
+  border-color: rgba(255, 255, 255, 0.5);
+  font-family: 'roboto', sans-serif;
+  font-weight: bolder;
+  font-size: 2rem;
+}
```

Comment:
- [ OPTIONAL ]  Great job but I think that If I scroll i will see all the sections but depend on the Figma template I should just see the menu without scrolling in the side so kindly fix it to match the Figma.🔥🔥💪🏻💪🏻✅